requirements.txt

```diff
@@ -61,7 +61,7 @@ pyparsing~=2.4.7
 pyrsistent~=0.17.3
 python-dateutil~=2.8.1
 pytz~=2020.1
-PyVISA~=1.11.1
+PyVISA~=1.10.1
```

Comment:
This one should be left as is. We still want to recommend and test with 1.11.1

Reply:
okay thanks I wasn't too sure about changing this. I'll revert.